microbenchmarks/results.txt

```diff
@@ -0,0 +1,89 @@
+# on macbook pro
```

Comment:
we probably don't need this file to be checked in.

Reply:
@sijie It takes awhile to get results, I assume we don't want to run these on each build.
OTOH having benchmark results easily accessible may help people decide on digest type to use. 
I can remove it, of course, if you insist.

Reply:
@dlg99 okay for me.

Reply:
I think it's good to have it somewhere. To help people decide, probably we should also mention a summary of the differences in the Javadoc and website.